packages/runtime/src/api/react.ts

```diff
@@ -19,21 +19,8 @@ import {
   CreateTableRecordMutationVariables,
 } from './graphql/generated/types'
 
-export type SerializedLocalizedResourcesMap = {
-  [resourceId: string]: string | null
-}
-
-type LocalizedResourcesMap = Map<string, string | null>
-
 export type CacheData = MakeswiftApiClient.SerializedState
 
-export type MakeswiftClientOptions = {
-  uri: string
-  cacheData?: CacheData
-  localizedResourcesMap?: SerializedLocalizedResourcesMap
-  locale?: Intl.Locale
-}
-
```

Comment:
These weren't/shouldn't used externally, collapsed into the constructor definition.